### ISSUE TITLE
-Ofast option prevents compilation

### ISSUE THREAD

**Issue by JunAishima** [kiyo-masui/bitshuffle#33]
Hello,

We are using gcc 4.4.7 under Centos 6.7 64-bit, and while I was trying to install the latest code (pulled this morning 2016-03-23) with "pip install bitshuffle", I got the following:

>gcc -pthread -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC ->DBSHUF_VERSION_MAJOR=0 -DBSHUF_VERSION_MINOR=2 -DBSHUF_VERSION_POINT=1 ->I/home/aishimaj/autodataset/lib/python2.7/site-packages/numpy/core/include -Isrc/ -Ilz4/ ->I/beamline/python/include/python2.7 -c bitshuffle/ext.c -o build/temp.linux-x86_64-2.7/bitshuffle/ext.o ->Ofast -march=native -std=c99 -fopenmp -fno-strict-aliasing
>    cc1: error: invalid option argument ‘-Ofast’
>    error: command 'gcc' failed with exit status 1

Another github issue shows that an alternative optimization argument can be used - https://github.com/taoliu/MACS/issues/91

I followed their directions for the change at the very bottom and it worked for me.

Basically, any configuration that does not require us to poke around with the setup file would be appreciated!

Jun Aishima
Australian Synchrotron/Monash University

**Comment by kiyo-masui** [comment]
Yes, fixing this should not be a problem. We can just replace it with '-O3 -ffast-math'.  If you can make that change and put in a pull request that would be great.

Note that the file "conda-recipe/setup.py.patch" also deals with this, so the PR should remove that part of the patch as well as the note in the Anaconda part of the README.
